libkbfs/kbfs_ops_test.go

```diff
@@ -337,14 +337,15 @@ func TestKBFSOpsGetRootNodeReIdentify(t *testing.T) {
 	require.NoError(t, err)
 	assert.True(t, fboIdentityDone(ops))
 
-	// The channel is not buffered, when the second value is received
-	// the first round of marking for reidentify will be done.
+	// Mark everything for reidentifying, and wait for it to finish
+	// before checking.
 	kop := config.KBFSOps().(*KBFSOpsStandard)
-	kop.reIdentifyControlChan <- struct{}{}
-	kop.reIdentifyControlChan <- struct{}{}
```

Comment:
I tried to explain it in the PR/commit description -- the second write was just a hacky way of knowing that the results of the first write were finished, but that had unintended consequences.